index.bs

```diff
@@ -258,7 +297,29 @@ so, HTML is patched as follows:
         :   For {{WorkerGlobalScope}} objects:
         ::  Return <var ignore>worker global scope</var>'s [=WorkerGlobalScope/embedder policy=].
 
-3.  The [$create a new browsing context$] algorithm sets the [=document/embedder policy=] for
+3. An <dfn>embeedder policy reporting endpoint</dfn> is a nullable string.
+
+4. The [=/embedder policy reporting endpoint=] is persisted on a number of objects:
```

Comment:
Similar to the above, it seems more reasonable to me to make the embedder policy a struct of the value and the reporting endpoint.

Reply:
Done.